### DISCUSSION TITLE
AerisWeather acquired by another company

### DISCUSSION THREAD

**Discussion by ehcloninger** [poblabs/weewx-belchertown#965] [General]
I'm doing a new install of weewx and I've been looking at the Belchertown skin for my own personal enjoyment. Nice job, BTW.

I was going through the detailed setup and looking to add the forecasts from AerisWeather using the docs' guidance. I created a PWS account and attached my data stream to it. Then, using the link on the Belchertown docs page, I started to create an account on AerisWeather. That's when I noticed the domain change to XWeather and the company name is Vaisala. I don't know anything about Vaisala, but it seems they are in the data collection/selling business.

https://www.xweather.com/blog/article/vaisala-xweather-x-aerisweather

I couldn't find the option to create a new project and honestly I'm a bit wary of these kinds of takeovers into larger entities. It seems there is a free tier for PWS contributors capped at 5,000 API hits per day. That's fine for a personal project, but if you're setting up a local hub, you might get API outages or end up with unexpected invoices.

**Comment by ehcloninger** [comment]
Bit of a followup. It seems that PWSWeather is also owned by Vaisala. I was able to create a new API key and secret from their dashboard and enable it in the skin to get the forecasts. It all seems to work. Since this is on a self-host behind Tailscale, it shouldn't cause me any problems, but if you're incorporating the skin into a public-facing IP, you might run up against your daily API limits.

**Comment by steepleian** [comment]
That is logical as PWS was also owned by AerisWeather. No issue with limits on public facing IP, you just need to cache the json data file that is retrieved so the XWeather API servers are not constantly being hit.

https://claydonsweather.org.uk
